### PR TITLE
Update UI colour

### LINKED ISSUE #1328: Rebrand: Update Kedro-viz

### PR DESCRIPTION
**Pull request by Huongg**
## Description

Resolves #1328 

## Development notes

Light theme option 1 [design link](https://www.figma.com/file/K50WL8AfBsuqHREHNJkbcM/Kedro-Design%3A-(IN)-UI-colour-update?type=design&node-id=7026-150374&t=zM2kqkjJ3mxPSwzK-0). Changes are to use the existing white palette with stroke lines
Dark theme option 1 [design link](https://www.figma.com/file/K50WL8AfBsuqHREHNJkbcM/Kedro-Design%3A-(IN)-UI-colour-update?type=design&node-id=6986-44319&t=R5OqzehJN6qAKUcp-0). Changes are to update all the existing slate palette

## QA notes
Things are missing:
- New kedro logo
- Minimap colour for dark theme
- For light theme in flow chart, do we need the blue line for active/selected node as there isn't one for the dark theme. 
- Release note

## Checklist

- [x] Read the [contributing](/CONTRIBUTING.md) guidelines
- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Updated the documentation to reflect the code changes
- [ ] Added new entries to the `RELEASE.md` file
- [ ] Added tests to cover my changes

<a href="https://gitpod.io/#https://github.com/kedro-org/kedro-viz/pull/1359"></a>

### CODE REVIEW

**Comment by Mackay031** [comment]
Hi, looking good so far. I have a couple of quick comments. I have taken screen shots and will add comments to each. This is a quick first round of comments of issues I immediately noticed. I will take a more in depth look today.

Screenshot 2023-05-16 at 11 10 03

The box in the hover state overlaps the keyline


Screenshot 2023-05-16 at 11 06 46

The background of the search section should be the TESTslate-600




Screenshot 2023-05-16 at 11 25 43

The background of the search section should be the white-100




Screenshot 2023-05-16 at 11 05 12

Can we make the blue highlight the same fill and stroke as the dark theme


Screenshot 2023-05-16 at 11 32 16

The group highlight state in the list is a darker colour. Can we make it slate-200
(#242526) to match the other highlight state



Screenshot 2023-05-16 at 11 39 07

Can we add a white-300 keyline at the bottom of the 'all' divider section and all other divider section. If the top run is selected as in this example, then there is no well defined differentiation. This is what I mentioned yesterday on the call.

Screenshot 2023-05-16 at 11 02 24


Has the code block always scrolled right i.e the code is cut off. Maybe be it as...


Thanks!

**Comment by Mackay031** [comment]
Regarding this point:

The background of the search section should be the white-100. I've checked on my code and in the browser, its white-100: #F9F9FA

Screenshot 2023-05-16 at 16 02 19

In the above screenshot from the Figma file, the 'search' background is the same shade as the list items. If the 'search' background is different, then maybe the list items and runs list background are not white-100 (#F9F9FA0? Could you check? Thanks :)

**Comment by Huongg** [comment]
hey @Mackay031, for the final point, its now fixed. They were all the same colour but there was a shadow box from the scrollbar in between thats why it makes it looks like it looks different. I've removed that shadow now.

**Review by rashidakanchwala — COMMENTED** [review]
I have only looked at flowchart in light theme so far. Here are some design deviations I found. 


1. there's some left and bottom padding in the design for minimap which is missing. 


Screenshot 2023-05-19 at 11 41 02

2. The color of show code [blue] is different from design. 


Screenshot 2023-05-19 at 11 39 57

3. The Tags and Parameters have a different color (darker shades of grey in design) 



Screenshot 2023-05-19 at 11 38 42

Screenshot 2023-05-19 at 11 37 33

**Comment by Huongg** [comment]
thanks @rashidakanchwala @Mackay031 , I believed I've addressed the combination of both of your feedback. There are some left that I need to discuss with @Mackay031 to double check with him. Let me know if you're free today or tomorrow to talk about it 😄

**Review by tynandebold — COMMENTED** [review]
The code looks good! No comment from me there.

I do have some comments on the colors, specifically the dark theme. On the screen I'm using at the office (ThinkVision monitor), it's hard for me to distinguish between certain elements on the page, those being:

- The nodes on the flowchart and the background
  - Previously our contrast ratio here was 1.6 and now it's 1.29, a ~20% decrease (doesn't matter that it's failing the test for text, as I'm using this more as a comparison of numbers)
    Screenshot 2023-05-23 at 14 38 14
- The modals (settings and edit exp. tracking run). There's barely any contrast there.
- In experiment tracking, it's hard to see any difference between the run metadata and run tracking data section as well as the tabs: 
    image

I want to make sure we're checking these colors on non-retina and non-4k screens. Are we? When I look at the update on my MacBook Pro I can see things much clearer, but we can't assume all or even most of our users are using that hardware.

Lastly, the light theme looks amazing, even on this monitor at the office. I've always liked it, and now like it even more.

cc @Mackay031 @stephkaiser

**Review by rashidakanchwala — APPROVED** [review]
Just reviewed the code, not the UI and it all looked fine. thanks Huong

File: src/components/experiment-tracking/parallel-coordinates/parallel-coordinates.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ const paddingLeftRight = 80;
 const axisGapBuffer = 3;
 const selectedMarkerRotate = [45, 0, 0];
 
-const selectedMarkerColors = ['#0084B2', '#FFBC00', '#31E27B'];
+const selectedMarkerColors = ['#A000BC', '#FFBC00', '#31E27B'];
```
[the remark this answers is not in the document]
Comment (Huongg) [reply src/components/experiment-tracking/parallel-coordinates/parallel-coordinates.js:27]: hey yeah good shout, I've moved them to css now so we can re-use the variable

**Review by tynandebold — APPROVED** [review]
The code LGTM as well.

**Comment by Huongg** [comment]
hey @tynandebold to confirm, @Mackay031 and I have addressed point 1 and 2 from your comments. Point 3 will be done in the future as it requires more thoughts and changes on that

**Pull request merged**